src/components/event/Event.jsx

```diff
@@ -35,6 +35,13 @@ const Event = ({ event, onLike }) => {
                   {themes.map(theme => <Link key={theme.id} to={`/theme/${theme.id}`}>{theme.title}</Link>)}
                 </p>}
             </Col>
+            <Col>
+              {tags && tags.length > 0 &&
+                <p className='lead mb-0'>
+                  <strong className='mr-3'>{t('Tags', { n: tags.length })}</strong>
+                  {tags.map(tag => <span key={tag.id} to={`/theme/${tag.id}`} className='mr-3'>{tag.title}</span>)}
```

Comment:
`/theme/${tag.id}` should probably be `/tag/${tag.id}`, but I'm not sure we actually have a “tag” list/detail view. Maybe we should just drop the link.

Reply:
Good catch! It should have been remove. I have committed the change.